### DISCUSSION TITLE
HTML and CSS connecting

### DISCUSSION THREAD

**Discussion by preguntas19** [community/community#21367] [Other Feature Feedback, Questions, & Ideas]
<p>Hi.</p>
<p>I am new in Gethub.</p>
<p>I saw that there are posts about CSS and HTML problemas, but I do not find the answer I am looking for.</p>
<p>I have a web page and CSS is not applied to html. When I made the smaller CSS I applied it. Can you explain the mistake I have? I did the validation of w3c but it says it is correct.</p>
<p>I attach 2 files: HTML and CSS.</p>
<p>Thanks. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/49c1d3df29d30040c819a7c2d904343e12ef5e19.jpeg" data-download-href="/uploads/short-url/awu80iYuhzAC4OANxnYgAKgQ2uJ.jpg?dl=1" title="gethubhtml.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/47985378/181063899-21fd4d50-00ec-435f-83eb-f3e71b0fbc68.jpg" alt="gethubhtml.JPG" data-base62-sha1="awu80iYuhzAC4OANxnYgAKgQ2uJ" width="690" height="440" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/49c1d3df29d30040c819a7c2d904343e12ef5e19_2_690x440.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/4/49c1d3df29d30040c819a7c2d904343e12ef5e19_2_1035x660.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/4/49c1d3df29d30040c819a7c2d904343e12ef5e19_2_1380x880.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/49c1d3df29d30040c819a7c2d904343e12ef5e19_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">gethubhtml.JPG</span><span class="informations">1795×1145 302 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/c/c7016425684b73ffe8f4dbf368eda8984d27f5ea.jpeg" data-download-href="/uploads/short-url/sotZPoPI4P6H6AE9eBV1HwfkB3c.jpg?dl=1" title="gethubcss.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/47985378/181063900-0e16bf57-3e8e-4356-8216-e9bef79115e3.jpg" alt="gethubcss.JPG" data-base62-sha1="sotZPoPI4P6H6AE9eBV1HwfkB3c" width="475" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/c/c7016425684b73ffe8f4dbf368eda8984d27f5ea_2_475x500.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/c/c7016425684b73ffe8f4dbf368eda8984d27f5ea_2_712x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/c/c7016425684b73ffe8f4dbf368eda8984d27f5ea_2_950x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/c7016425684b73ffe8f4dbf368eda8984d27f5ea_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">gethubcss.JPG</span><span class="informations">1449×1524 140 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by etseal** [comment]
<p>make sure u have put your css code in the proper places in the &lt;head&gt; of your document…</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>You’ve mixed up classes and ids in your css.</p>
<p>E.g. if “navbar” is a class as shown in the HTML, then the CSS should read “.navbar”, not “<span class="hashtag">#navbar</span>”.</p>

**Comment by preguntas19** [comment]
<p>Hi. Thanks for your answer, but I applied your recommendations and the css still does not apply to html. The verification of w3c says that the code is correct. Can you help me, please?</p>
<p>I attach html and css new files. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/2b07ee253f415afe83d75a3b825360b5bcbd2419.jpeg" data-download-href="/uploads/short-url/68FwlvsAhY04j3fYlZlbq12OcuR.jpg?dl=1" title="gethub12-3.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/47985378/181063905-ff7e6595-6949-4ed4-bd19-50415f62de68.jpg" alt="gethub12-3.JPG" data-base62-sha1="68FwlvsAhY04j3fYlZlbq12OcuR" width="343" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/2b07ee253f415afe83d75a3b825360b5bcbd2419_2_343x500.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/2/2b07ee253f415afe83d75a3b825360b5bcbd2419_2_514x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/2/2b07ee253f415afe83d75a3b825360b5bcbd2419_2_686x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/2b07ee253f415afe83d75a3b825360b5bcbd2419_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">gethub12-3.JPG</span><span class="informations">1128×1641 143 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/4350d06f60ed90ad24a2a8c4e6607d41ac86b67f.jpeg" data-download-href="/uploads/short-url/9Bv8jEgH53bFTd9Zr44vVQ2uzv1.jpg?dl=1" title="gethubhtml3-12.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/47985378/181063907-25a28a6e-d4db-444c-8912-02011e6cfcb4.jpg" alt="gethubhtml3-12.JPG" data-base62-sha1="9Bv8jEgH53bFTd9Zr44vVQ2uzv1" width="512" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/4350d06f60ed90ad24a2a8c4e6607d41ac86b67f_2_512x500.jpg, https://aws1.discourse-cdn.com/github/optimized/2X/4/4350d06f60ed90ad24a2a8c4e6607d41ac86b67f_2_768x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/4/4350d06f60ed90ad24a2a8c4e6607d41ac86b67f_2_1024x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/4350d06f60ed90ad24a2a8c4e6607d41ac86b67f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">gethubhtml3-12.JPG</span><span class="informations">1620×1579 403 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by naturaleza106** [comment]
<p>7 &lt;link rel=”stylesheet” type=”text/css” href=”Mystyles.css media=”screen”/&gt;<br>
te sobra la barra que tienes antes de Mystyles.css</p>

**Comment by naturaleza106** [comment]
<p>8 &lt;link rel=”stylesheet” type=”text/css” href=”Mystyles.css media=”screen”/&gt;<br>
te sobra la barra que tienes antes de Mystyles.css</p>

**Comment by preguntas19** [comment]
<p>Gracias, pero antes de Mystyle.css no hay barra, hay comillas. Te refieres a eso?</p>
<p>Por otra parte, si sólo tengo un archivo css que es colores.css debo poner todo ese link diciendo Mystyles.css (lo copié de un tutorial) o es justamente donde debo reemplazar por mi archivo css?</p>

**Comment by naturaleza106** [comment]
<p>prueba si consigues que funcione, sin la barra, de todos modos, creo que lo que tines que hacer es realizar un enlace con el css original que has de tener puesto en tu ordenador y dentro de la carpeta en la que tienes almacenado el programa, no se bien lo que has hecho, pero es lo que necesitas, a parte de que en css al comienzo de alguna haccion no lleva barras invertidas.</p>
<p>un saludo</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/naturaleza106">@naturaleza106</a>,</p>
<p>Thanks a lot for being here! At this time we don’t have moderators to support a different language than English. I see this thread has been very informative and helpful to all and will ask you to please resume the conversation in English. Thanks again for taking the time to contribute, we look forward to seeing you around!</p>
<p>Best,</p>
<p>Andrea</p>

**Comment by nabilateuse** [comment]
<p>Hi,</p>
<p>You’ve mixed up classes and ids in your css.<br>
the CSS should read “.navbar”, not “<span class="hashtag">#navbar</span>” and ‘.principal’ ,not "<span class="hashtag">#principal</span>’.</p>

**Comment by hassan647** [comment]
<p>(do not write a class with an  # (hash) but instead write it with an  .  (dot) while writing a class in css </p>
